src/data.json

```diff
@@ -0,0 +1,411 @@
+{
+    "LoginPageData" : {
```

Comment:
Some of these property names are a bit too verbose for my taste, but then again, that's mostly irrelevant for the browser's JS engine, so no particular action item required, just stating a personal opinion. :)